odoo/addons/base/models/ir_actions.py

```diff
@@ -642,7 +762,43 @@ def _run_action_object_write(self, eval_context=None):
             for field, new_value in res.items():
                 record_cached[field] = new_value
         else:
-            self.env[self.model_id.model].browse(self._context.get('active_id')).write(res)
+            starting_record = self.env[self.model_id.model].browse(self._context.get('active_id'))
+            _, _, target_records = self._traverse_path(record=starting_record)
+            target_records.write(res)
+
+    def _run_action_webhook(self, eval_context=None):
+        """Send a post request with a read of the selected field on active_id."""
+        record = self.env[self.model_id.model].browse(self._context.get('active_id'))
+        url = self.webhook_url
+        if not url:
+            raise UserError(_("I'll be happy to send a webhook for you, but you really need to give me a URL to reach out to..."))
+        vals = {
+            '_model': self.model_id.model,
+            '_id': record.id,
+            '_action': '%s(#%s)' % (self.name, self.id),
```

Comment:
Remy said I coded like fp (😱 )because i didn't use f-strings... I guess that's what happens when you develop once in a blue moon: you don't follow technical progress anymore 😢